src/main/java/org/breedinginsight/services/validators/TraitValidatorError.java

```diff
@@ -69,8 +69,23 @@ public ValidationError getMissingScaleDataTypeMsg() {
     }
 
     @Override
-    public ValidationError getMissingTraitNameMsg() {
-        return new ValidationError("traitName", "Missing trait name", HttpStatus.BAD_REQUEST);
+    public ValidationError getMissingObsVarNameMsg() {
+        return new ValidationError("observationVariableName", "Missing observation variable name", HttpStatus.BAD_REQUEST);
+    }
+
+    @Override
+    public ValidationError getMissingTraitEntityMsg() {
+        return new ValidationError("entity", "Missing trait entity", HttpStatus.BAD_REQUEST);
```

Comment:
Relating to previous comment, if the ObservationVariable was changed to have a trait object property then these would be trait.entity, trait.attribute, and trait.traitDescription to be consistent with how the other messages are formatted with the first part being the brapi object the properties are under. But that could be something done in another card.

Reply:
agreed. There needs to be a major refactoring done(separate card) on the backend to go from using Trait as the highest level object (as it has been) to using observation variable as the highest level object(as you describe) to be consistent with the brapi model.
The scope of this work is functinality: making the changes necessary to get the backend to support the MVP changes made on the frontend. In the spirit of breaking the work up into manageable pieces done in a single sprint, I think the remaining necessary refactoring should be a separate card.

Reply:
sounds good, could you create the card for that or at least a placeholder for it